src/atopile/cli/create.py

```diff
@@ -3,8 +3,8 @@
 import re
 import sys
 import textwrap
-import webbrowser
 from pathlib import Path
+import webbrowser
```

Comment:
nit; (wouldn't bother except there's two other linting errors blocking the PR)
Run "Organise Imports" command in VSCode, I believe this shouldn't have moved (and can impact merges if someone else has touched imports)

Reply:
fixed